BUILD_VERSIONS

```diff
@@ -2,3 +2,4 @@
 export SYNAPSE_VERSION=v1.5.1
 export POSTGRES_VERSION=10
 export NGINX_VERSION=1.17
+export RAIDEN_VERSION=v0.200.0-rc1
```

Comment:
nit: Latest version I see is 0.200.0-rc2, but we need to update this later anyways.